main/static/script.js

```diff
@@ -65,6 +65,42 @@ module.directive("googleChart", function($chartWaiting, $timeout, $windowSize) {
         }
         render();
       });
+      function unitless(value) {
+        if (typeof value !== "string") {
+          return value;
+        }
+        if (value.substring(value.length-2) == "px") {
```

Comment:
rather to regex match for `px$` and `%$` - i think they're more concise and communicating (no need to do subtraction).